docs/UserGuide.adoc

```diff
@@ -388,7 +388,9 @@ Moves the text at index 3 to the right by 20% of the image width and up by 30% o
 
 [TIP]
 You can type `move INDEX` and use arrow keys to move the specified text. +
-Hold kbd:[Shift] while pressing arrow keys to make the text move faster.
+Hold kbd:[Shift] while pressing arrow keys to make the text move faster. +
+Hold kbd:[Alt] while pressing arrow keys to make the text move slower.
```

Comment:
Might want to specify what Alt is on MacOS (which is Option key).

Reply:
Have you tried if Option key works?

Reply:
Yes, the increment is reduced. Option behaves like Alt on MacOS.